### DISCUSSION TITLE
Change lightbox icons

### DISCUSSION THREAD

**Discussion by samuelcerezo** [elementor/elementor#23442] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It would be great to have the ability to change the lightbox icons. Many times, the icons in lightbox do not correspond to the family of icons I've used in the rest of the web.

### Describe the solution you'd like

Possibility to upload an svg or select the icon for the close, left, right, zoom, share... buttons.

**Comment by Tjarkina** [comment]
Because I do not like the style of the icons in the lightbox AND more importantly it does not meet the 100% customizability that Elementor has claimed to be for years ... I do have to support this request. 

And there are a few bugs in the lightbox feature. The size and color of the navigation icons will not change. This means I can not use a light background because of the light grey arrows left and right; you don't see them (good). It's all about contrast and here it is impossible to add some contrast.
